.github/project.yml

```diff
@@ -1,4 +1,4 @@
 name: SmallRye Config
```

Comment:
Config?

Reply:
I guess it was a copy from there, but it has no effect on the release. Regardless, it should be fixed.

Reply:
Classic copy paste, Roberto first set this up for config, so I guess this just never got updated when porting. 

Opened https://github.com/smallrye/smallrye-opentelemetry/pull/414